lib/tasks/export.rake

```diff
@@ -0,0 +1,34 @@
+require 'csv'
+
+namespace :reports do
+  task :export => :environment do
```

Comment:
Please add rake comment, something like "Export all reports as CSV files. This task is only for development purposes."